### ISSUE TITLE
CSS to SASS – cleanup

### ISSUE THREAD

**Issue by jorgenevil** [systemapic/systemapic.js#110]
Clean up all CSS as move to SASS files.

**Comment by knutole** [comment]
Do we really need SASS files? They're so hard to work with (ie. debug)? 

What's the pros'n cons for SASS? Could we not just use CSS?

**Comment by jorgenevil** [comment]
Yes, we need SASS files. They shout be possible to debug, as they should come with a map to explain which file it's from. I can look into why it's only working some times.

The pros with SASS is that you can store global variables in them, such as colors, fonts etc, and change them in one place.

**Comment by jorgenevil** [comment]
Woop woop!

I fucked it up :P – We currently have `node-sass` version `2.1.1` installed, and I'd like to update it to current version, which is `3.4.2` –– that version will give us nice source maps, that will show the origin of the file when debugging in chrome.

HOWEVER, when _tried_ to install it on the BETA, shit got fucked up, and now the old SASS compiler doesn't work: `>> Error: Cannot find module 'node-sass'`

I did this in the `/docks/modules/wu/` folder...

Node-sass: https://github.com/sass/node-sass
Sourcemaps: http://www.sitepoint.com/using-source-maps-debug-sass-chrome/

**Comment by knutole** [comment]
Har alltid vært mye bugs med `node-sass`. Det hjelper ikke å installere noe på host, du må installere inni container. Men - jeg synes nå det er mye mas for liten effekt. Det er jo ikke sånn at vi bruker globals så mye uansett? Font, farger, etc. blir jo satt på alle mulige steder, og sånn har det jo alltid vært?  Men ikke mitt bord..

**Comment by jorgenevil** [comment]
Actually, we don't really need the new version. All it takes to show the origin of the SASS file, is to click the "Auto-reload generated CSS" in Chrome settings, and it will show you the origins.

![screen shot 2016-02-17 at 7 00 21 pm](https://cloud.githubusercontent.com/assets/8961926/13119310/c83f7e78-d5a8-11e5-87ef-9454ea66d8a2.png)

**Comment by jorgenevil** [comment]
Jeg mener det å gå bort fra SASS er å gå i feil retning. Jeg vil ha SASS :P

**Comment by jorgenevil** [comment]
Så, hvordan er det jeg resetter NPM modulene til slik det var før jeg kåla det til? Kjøre `npm install` i en eller annen folder?

**Comment by jorgenevil** [comment]
@knutole ?

**Comment by jorgenevil** [comment]
Er det sånn at du bare har bestemt at vi ikke skal ha support for SASS lenger, og lar det bli med det?
